fhirResourcesToLoad/o2jane_04__device-requestB.json

```
@@ -23,8 +23,8 @@
   "codeCodeableConcept": {
     "coding": [
       {
-        "system": "http://www.ama-assn.org/go/cpt",
-        "code": "95259",
+        "system": "https://bluebutton.cms.gov/resources/codesystem/hcpcs",
+        "code": "E0607",
         "display": "Home Blood Glucose Monitor"
```

Suggested to match the display value defined for this code. Found in https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1219.84/definition.
```suggestion
        "display": "Home blood glucose monitor"
```